Makefile

```diff
@@ -6,20 +6,21 @@
 #    By: floris <marvin@42.fr>                        +#+                      #
 #                                                    +#+                       #
 #    Created: 2020/06/11 12:44:37 by floris        #+#    #+#                  #
-#    Updated: 2020/09/10 12:43:56 by merlijn       ########   odam.nl          #
+#    Updated: 2020/09/11 20:27:42 by mvan-eng      ########   odam.nl          #
 #                                                                              #
 # **************************************************************************** #
 
-VM_LOC = ./srcs/vm
-VM_NAME = $(VM_LOC)/corewar
+VM_LOC = ./srcs/vm/
+VM_NAME = corewar
 
 ASM_LOC = ./srcs/asm
 ASM_NAME = $(ASM_LOC)/asm
 
 all: $(VM_NAME) $(ASM_NAME)
 
-$(VM_NAME):
+corewar:
```

Comment:
Waarom is dit niet meer $(VM_NAME)

Reply:
Was aangepast omdat de tester anders niet werkte, kan worden teruggezet